app/authenticated/project/controller.js

```diff
@@ -5,7 +5,7 @@ export default Ember.Controller.extend({
   prefs: Ember.inject.service(),
 
   tags: '',
-  byStack: '',
+  byStack: 1,
```

Comment:
Numerical `1` instead of string `"1"` seems likely to cause issues with not matching later